lib/request_handler/concerns/config_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module RequestHandler
+  module Concerns
+    module ConfigHelper
+      def lookup!(hash, key)
+        hash.dig(*symbolize_key(key)).tap do |data|
+          raise NoConfigAvailableError, key.to_sym => 'is not configured' if data.nil?
```

Comment:
Why not return nil here instead of slow exceptions? This is called e.g. on every multi part request

Reply:
We sometimes want to raise exceptions if a config is missing and can't ignore it.

Reply:
would `lookup(hash, key) || raise ...` be sufficient? I guess it would not handle the case when the value is `false` correctly but not sure we even need this case